pom.xml

```diff
@@ -57,18 +60,14 @@
     </scm>
     <issueManagement>
         <system>GitHub</system>
-        <url>https://github.com/SpigotMC/BungeeCord/issues</url>
+        <url>https://github.com/Letsmine/BungeeCord/issues</url>
     </issueManagement>
-    <ciManagement>
-        <system>jenkins</system>
-        <url>http://ci.md-5.net/job/BungeeCord</url>
-    </ciManagement>
 
     <properties>
         <build.number>unknown</build.number>
         <netty.version>4.1.9.Final</netty.version>
-        <maven.compiler.source>1.7</maven.compiler.source>
```

Comment:
nope, why increase java version and removing compability without mentioning?
and without using java 8 features?